### PR TITLE
Preparing release of Spark v0.1.0-beta.0

### PR DESCRIPTION
**Pull request by TobiasFeld22**
Okay, we're currently working on almost releasing v0.1.0 to the public.
Before this happens a few final things need to happen:
- Add right versions to the package.json
- Review all code.
- Start sending out beta versions
- Implementing help command (#48)
- Compacting code, useless whitelines that don't help with readability (2 lines or more are too much)
- Beautify ugly code.

### CODE REVIEW

**Comment by TobiasFeld22** [comment]
@Sanil14 @sharletp @advaith1 Please review the code now ;)

File: commands/reload.js [hunk 1]
```diff
@@ -1,304 +1,100 @@
...
+    reloadCommands();
+    await reloadSearch();
+    return message.channel.send("Successfully reloaded all commands.");
+  } else if(message.content.split(" ")[1]  === "observers") {
+    message.channel.send("Reloading all observers...");
+    reloadObservers();
+    await reloadSearch();
+    return message.channel.send("Successfully reloaded all observers.");
+  } else if(message.content.split(" ")[1] === "engines") {
+    message.channel.send("Reloading all enginens...");
```
Comment (advaith1) [inline commands/reload.js]: Typo

Comment (TobiasFeld22) [reply commands/reload.js]: Fixed

File: README.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
-# Spark
-###### Javascript framework for Discord bots based on discord.js
----
-This is a small project made by [Tobias#5233](https://discordapp.com/) that should make it easy to create a bot.  
+![header](https://cdn.discordapp.com/attachments/237019716546199552/432200563485179906/banner-outlined-no-bg4x.png)
```
Comment (advaith1) [inline README.md:1]: Spark image has the blue dots

Comment (TobiasFeld22) [reply README.md:1]: It's supposed to have blue dots

Comment (advaith1) [reply README.md:1]: Oh I thought it was an issue with the SVG, since the site header had blue dots and you switched it to the PNG which doesn't.

Comment (TobiasFeld22) [reply README.md:1]: That one also has blue dots, it's just very tiny and compressed

File: commands/help.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        var data = await getData();
+        if (message.channel.permissionsFor(message.guild.members.get(client.user.id)).serialize().EMBED_LINKS) {
+            embed.setTitle(`${client.user.username} help information`)
+            embed.setDescription(`Type ${client.config.prefix[0]}help command-name to get more information.`)
+            data.forEach((entry) => {
+                var {command} = entry;
+                return embed.addField("• " + command.name, command.description, false)
+            })
+            embed.setColor(client.config.embedColor || 0xffe13f)
+            embed.setFooter(data.commands.size + " commands loading")
```
Comment (advaith1) [inline commands/help.js]: Default `help` command doesn't work and throws an error:
```
(node:17789) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'size' of undefined

    at sendCommands (/app/node_modules/sparkbots/commands/help.js:50:43)

    at <anonymous>

(node:17789) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
```

**Comment by TobiasFeld22** [comment]
@advaith1 looking good 👌

**Review by Sanil14 — APPROVED** [review]
Prominent errors have been fixed. Everything looks good so far!

**Pull request merged**